getting-started/kubernetes/managed-public-cloud/eks.md

```diff
@@ -11,13 +11,53 @@ Enable Calico in EKS managed Kubernetes service.
 
 EKS has built-in support for Calico, providing a robust implementation of the full Kubernetes Network Policy API. EKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the Calico Network Policy API.
 
+You can also use Calico for networking on EKS in place of the default AWS VPC networking. This allows you to take advantage of the full set of Calico networking features, including Calico's flexible IP address managemenet capabilities.
+
 ### How to
 
-To enable Calico network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Installing Calico on Amazon EKS' url='https://docs.aws.amazon.com/eks/latest/userguide/calico.html' %}
+#### Using VPC networking
```

Comment:
Minor, but in general I think we've aimed for titles where you can put the words "how to" in front. Not sure if we've done this everywhere or not though. So maybe "Install EKS with Amazon VPC networking" and "Install EKS with Calico networking"?